frontend/src/features/comparisonSeries/ComparisonSeries.tsx

```diff
@@ -215,6 +220,24 @@ const ComparisonSeries = ({
     setDialogOpen(false);
   };
 
+  const skipTutorial = () => {
+    closeDialog();
+
+    setTutorialIsSkipped(true);
+    window.localStorage.setItem('tutorialIsSkipped', 'true');
+  };
+
+  const skipTutorialButton = (
+    <Button
+      sx={{ marginRight: '8px' }}
```

Comment:
We can simplify the CSS by completely removing the `sx` here.

The button is displayed in a flex container, so we just have to specify a `gap` to define space between all flex items.

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1470/commits/99f51a1341a29c60a6219babba2bb194f1963c55